packages/gdl-frontend/pages/books/translations.js

```diff
@@ -333,9 +336,97 @@ const CustomGrid = styled('div')`
   }
 `;
 
-class MyTranslationsPage extends React.Component<{
-  intl: intlShape
-}> {
+const CustomSearchBar = styled('div')`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${TABLET_BREAKPOINT}px) {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+`;
+
+const CustomSearchbarItem = styled('div')`
+  width: 100%;
+  padding-top: 5px;
+`;
+
+function sortOnTitle(dir) {
+  return function(a, b) {
+    const result =
+      a.to['title'] < b.to['title']
+        ? -1
+        : a.to['title'] > b.to['title']
+        ? 1
+        : 0;
+    return result * dir;
+  };
+}
+
+function showSelectedTranslations(
+  translations,
+  refetch,
+  titleSearch,
+  sortDirection
+) {
+  if (sortDirection === 'asc') translations.sort(sortOnTitle(1));
```

Comment:
Her gjør du en direkte sortering på `translations`, som innebærer en mutering av objektet. Jeg pleier å sjekke opp i [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) for å dobbeltsjekke returverdier. I de fleste tilfeller ønsker mann ikke å mutere på objekter, fordi det kan føre til:
1. Muterer du en prop/state i React så vil den ikke skjønne at verdien har endret seg og da skjønner den ikke at den skal rerendre komponenten eller ikke.
2. Muterer du objektet her, så betyr det at alle andre steder vi bruker `translations` få en ny, kanskje uønsket `translations` og det er uheldig.